Procfile

```diff
@@ -1,4 +1,4 @@
-web: ./script/server-pgbouncer
+web: ./script/server-pgbouncer && ./script/appsignal
```

Comment:
AppSignal did get installed, so I am pretty sure the script ran. But since the installer is a one time thing, that does not block, I would put that before the pgbouncer script.